### DISCUSSION TITLE
Win UI 3 项目生成的一堆语言文件夹如何像 WPF 一样用配置只留下一个？

### DISCUSSION THREAD

**Discussion by Sealt** [BYJRK/DotNet-Discussions#221] [其他框架]
在 WPF + NET CORE 项目中可以使用这条配置处理那些语言文件夹
```xml
<SatelliteResourceLanguages>zh-CN</SatelliteResourceLanguages>
```

但是我在 Win UI 3 项目中无法使用，并且生成了如图 89 个语言文件夹，强迫症非常不适……
![image](https://github.com/BYJRK/DotNet-Discussions/assets/13269355/f3af7ed3-388e-4593-b21c-7232246045ff)

我尝试查阅官方文档
修改 Package.appxmanifest
```
  <Resources>
	  <Resource Language="zh-CN"/>
	  <Resource Language="en-us"/>
  </Resources>
```
修改 app.manifest
```
<assemblyIdentity version="1.0.0.0" name="WorkApp.app" language="zh-CN"/>
```
修改项目属性
```
<NeutralLanguage>en-US</NeutralLanguage>
```
询问 GPT3.5 和 BingAI，但都没有得到有用的结果……故来请教UP

**Comment by BYJRK** [comment]
这些文件夹如果确定删除也没有影响的话，要不然考虑写一个生成后事件？我以前也遇到过一个类似的情况，而且还是在WPF项目中，csproj里面加了那句也没用，很神奇。最后只能写了个脚本用来编译后删掉那些文件夹
